subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -1875,6 +1921,11 @@ SHELL_STATIC_SUBCMD_SET_CREATE(wifi_cmd_ap,
 		  "<MAC address of the station>\n",
 		  cmd_wifi_ap_sta_disconnect,
 		  2, 0),
+	SHELL_CMD_ARG(config, NULL,
+		  "Configure AP parameter.\n"
```

Comment:
`Configure AP, maximum STA inactivity timer`

Reply:
As there are multiple parameters to be configure, it's better to keep Configure AP parameter.